src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -383,10 +385,11 @@ public function setOptions(array $options)
      * @see http://github.com/drak/NativeSession
      *
      * @param AbstractProxy|NativeSessionHandler|\SessionHandlerInterface|null $saveHandler
+     * @param bool                                                             $updatePhpHandler
      *
      * @throws \InvalidArgumentException
      */
-    public function setSaveHandler($saveHandler = null)
+    public function setSaveHandler($saveHandler = null, bool $updatePhpHandler = true)
```

Comment:
adding a new argument to a public method is a BC break (breaks child classes),

Reply:
Good point 🤦‍♂️

Reply:
@chalasr updated 👍